PlacesSearchBar/AutoCompletePrediction.cs

```diff
@@ -89,7 +89,7 @@ public static AutoCompletePrediction FromJson(JObject json)
 			var r = new AutoCompletePrediction
 			{
 				Description   = json["description"].Value<string>(),
-				ID            = json["id"].Value<string>(),
+				// This throws NULL exception now. ID            = json["id"].Value<string>(),
```

Comment:
It's always better to remove code rather than comment it out, to keep things clean. A better approach to this problem however would be to replace the raw json parsing with a serializable data model, and allow nullability. This would improve things if google decide to further change the response in future.

Reply:
Is there a way to implement this nugget with this change made by razzledazzle013?